docs/source/_toctree.yml

```diff
@@ -1,68 +1,70 @@
-    title: Building a Browser Extension
-  - local: tutorials/electron
-    title: Building an Electron Application
-  - local: tutorials/node
-    title: Server-side Inference in Node.js
+    - local: tutorials/react
+      title: Building a React Application
+    - local: tutorials/next
+      title: Building a Next.js Application
+    - local: tutorials/sveltekit
+      title: Building a SvelteKit Application
+    - local: tutorials/browser-extension
+      title: Building a Browser Extension
+    - local: tutorials/electron
+      title: Building an Electron Application
+    - local: tutorials/node
+      title: Server-side Inference in Node.js
   title: Tutorials
 - sections:
-  - local: guides/private
-    title: Accessing Private/Gated Models
-  - local: guides/node-audio-processing
-    title: Server-side Audio Processing in Node.js
+    - local: guides/private
+      title: Accessing Private/Gated Models
+    - local: guides/node-audio-processing
+      title: Server-side Audio Processing in Node.js
   title: Developer Guides
 - sections:
-  - local: api/transformers
-    title: Index
-  - local: api/pipelines
-    title: Pipelines
-  - local: api/models
-    title: Models
-  - local: api/tokenizers
-    title: Tokenizers
-  - local: api/processors
-    title: Processors
-  - local: api/configs
-    title: Configs
-  - local: api/env
-    title: Environment variables
-  - sections:
-    - local: api/backends/onnx
-      title: ONNX
-    title: Backends
-    isExpanded: false
-  - sections:
-    - local: api/utils/core
-      title: Core
-    - local: api/utils/hub
-      title: Hub
-    - local: api/utils/image
-      title: Image
-    - local: api/utils/audio
-      title: Audio
-    - local: api/utils/tensor
-      title: Tensor
-    - local: api/utils/maths
-      title: Maths
-    - local: api/utils/generation
-      title: Generation
-    - local: api/utils/data-structures
-      title: Data Structures
-    title: Utilities
-    isExpanded: false
+    - local: api/transformers
+      title: Index
+    - local: api/pipelines
+      title: Pipelines
+    - local: api/models
+      title: Models
+    - local: api/tokenizers
+      title: Tokenizers
+    - local: api/processors
+      title: Processors
+    - local: api/configs
+      title: Configs
+    - local: api/env
+      title: Environment variables
+    - sections:
+        - local: api/backends/onnx
+          title: ONNX
+      title: Backends
+      isExpanded: false
+    - sections:
+        - local: api/utils/core
+          title: Core
+        - local: api/utils/hub
+          title: Hub
+        - local: api/utils/image
+          title: Image
+        - local: api/utils/audio
+          title: Audio
+        - local: api/utils/tensor
+          title: Tensor
+        - local: api/utils/maths
+          title: Maths
+        - local: api/utils/generation
+          title: Generation
+        # - local: api/utils/data-structures
+        #   title: Data Structures
```

Comment:
sorry I had to comment this to be able to run `npm run docs-preview`

Reply:
Did you run `npm run docs-api`? It should create the necessary files.

Reply:
ohh that's right, I've missed, that. :) I'll revert this. sorry for the yaml formatting then

Reply:
no worries haha